projects/vdk-core/src/vdk/internal/builtin_plugins/run/cli_run.py

```diff
@@ -143,14 +144,41 @@ def __log_exec_result(self, execution_result: ExecutionResult) -> None:
 
     @staticmethod
     def __log_short_exec_result(execution_result):
+        def extract_relevant_lines(step):
+            out = []
+            if step.exception:
+                call_list = traceback.format_tb(step.exception.__traceback__)
+
+                for i, line in enumerate(call_list):
+                    # Check if the step name is in the line
+                    if step.name in line:
+                        out.append(line)
+                        j = i + 1
```

Comment:
Please don't use j and i. 

Use sensible names that say what is j and i. Even Index or line_index, call_index or stack_depth are better.